simplq/src/components/pages/Admin/AdminSidePanel.jsx

```diff
@@ -8,7 +8,7 @@ import QueueDetails from '../Status/QueueDetails';
 
 export default (props) => (
```

Comment:
Destructure props in any react functional component. It is more obvious what do you expect, and you can use PropTypes to improve future coding.

```js
export default ({ queueId }) {
  dispatch(getQueue(queueId))
}
```